pkgs/development/compilers/go/binary.nix

```diff
@@ -11,6 +11,7 @@ let
     "armv6l" = "armv6l";
     "armv7l" = "armv6l";
     "powerpc64le" = "ppc64le";
+    "mips64el" = "mips64le";
```

Comment:
I don't understand why this is being added? It's for the binary bootstrap which doesn't seem to exist for this arch?

Reply:
sorry about that; fixed in new push.